internal/handlers/tigris/msg_update.go

```diff
@@ -198,7 +198,7 @@ func (h *Handler) MsgUpdate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 // update updates documents by _id.
 func (h *Handler) update(ctx context.Context, sp fetchParam, doc *types.Document) (int, error) {
 	id := must.NotFail(doc.Get("_id"))
-	f := must.NotFail(filter.Eq("_id", id).Build())
+	f := must.NotFail(json.Marshal(tigrisdb.Eq("_id", id)))
```

Comment:
@AlekSi do you want this to be something like `fmt.Sprintf(`{"_id": {"$eq": %s}`, id)`?

Reply:
`json.Marshal(map[string]any{...})`

Reply:
Done